src/LcdMenu.h

```diff
@@ -293,6 +288,11 @@ class LcdMenu {
      * ## Public Fields
      */
 
+    /**
+     * Cursor position
+     */
+    uint8_t cursorPosition = 1;
+    uint8_t previousCursorPosition = 1;
```

Comment:
You've already created a PR #80 which has a setter for `cursorPosition` so is there still a need to make it public, given that there's also a getter?

Reply:
You're right, I was already using `getCursorPosition`, and I don't need `previousCursorPosition`, so I'll close this.